virt/virt-4-10-release-notes.adoc

```diff
@@ -113,7 +113,7 @@ Removed features are not supported in the current release.
 
 * The VM Import Operator has been removed from {VirtProductName} with this release. It is replaced by the link:https://access.redhat.com/documentation/en-us/migration_toolkit_for_virtualization/2.2[Migration Toolkit for Virtualization].
 
-* This release removes the template for CentOS Linux 8. CentOS Linux 8 is no longer supported as of December 31, 2021.
+* This release removes the template for CentOS Linux 8. CentOS Linux 8 is no longer supported as of December 31, 2021. However, CentOS Stream 8 can be used with community support.
```

Comment:
I don't think this will be interpreted correctly as it's currently written. It also leaves out the fact that CentOS Stream 9 is provided in OCP 4.10 along with CentOS Stream 8. 

As written, someone might interpret this to mean the following: “Previously, we provided a template for CentOS Linux 8, which we (Red Hat) supported. Red Hat stopped supporting CentOS Linux 8 on December 31, 2021. However, you can now use CentOS Stream 8 instead (unclear if RH is providing this template or not) but it’s not supported by Red Hat.”

I propose the following rewrite:
```
* This release removes the template for CentOS Linux 8, which reached link:https://www.centos.org/centos-linux-eol/[End of Life (EOL)] on December 31, 2021. However, {product-title} now includes templates for CentOS Stream 8 and CentOS Stream 9.
+
[NOTE]
====
All CentOS distributions are community-supported.
====
```

cc @djfjeff

Reply:
The rewrite looks good to me, thanks @ousleyp !

Reply:
Yes, thanks for the elaboration @ousleyp!